pkg/azurefile/nodeserver.go

```diff
@@ -444,3 +456,19 @@ func makeDir(pathname string) error {
 	}
 	return nil
 }
+
+func checkGidPresentInMountFlags(volumeMountGroup string, mountFlags []string) (bool, error) {
+	gidPresentInMountFlags := false
+	if volumeMountGroup != "" {
+		for _, mountFlag := range mountFlags {
+			if strings.HasPrefix(mountFlag, "gid") {
+				gidPresentInMountFlags = true
+				kvpair := strings.Split(mountFlag, "=")
+				if !strings.EqualFold(volumeMountGroup, kvpair[1]) {
```

Comment:
there could be panic, use: 
`if len(kvpair) == 2 && !strings.EqualFold(volumeMountGroup, kvpair[1]) { }`

Reply:
write a ut for this function

Reply:
Done